### ISSUE TITLE
Never Scroll

### ISSUE THREAD

**Issue by 7flash** [karlicoss/HPI#228]
I found myself being trapped into social networks 

I have this idea 

i want to have this application that generates for me gives me all updates from creators I follow and topics I am interested in particularly

personal messages

new books - new movies - new games - new songs

new technology - new gadgets 

new science - new research - new 

- personal messagesasdf

**Comment by seanbreckenridge** [comment]
Currently the way this gets youtube data is through [google takekout](https://takeout.google.com/?pli=1), (can see the parser for it [here](https://github.com/seanbreckenridge/google_takeout_parser)), which is typically done on a monthly or so basis. That does have a list of your subscriptions, but it doesnt have creator data, its just what you've already watched.

Theres no patreon module here, thats also sort of creator focused.

Both of these could be achieved through new modules, it would likely be some script that runs in the background like one of these exporters:

- https://github.com/karlicoss/rexport
- https://github.com/karlicoss/ghexport
- https://github.com/seanbreckenridge/traktexport

But those as well, more focus on your own data -- videos you've watched, posts you've made, rather than what all of your followers are doing

So, to do what you want, would have to create two separate sort of projects, one for youtube, one of patreon, which downloads all of the data from people you're following. All possible, there just hasn't been a desire to do any of it yet, and personally feels a bit out of scope of HPI -- HPI helps you organize your own data, not data from people youre following (though sometimes, it does have that data). I may be wrong in my assumption here though, feel free to correct me @karlicoss 

To do the daily updates it would probably be some sort of script you run periodically/put in cron to send an email or something like that

**Comment by seanbreckenridge** [comment]
Personally I would probably solve this with RSS, but may not be the kind of experience youre looking for

**Comment by karlicoss** [comment]
Yup, in addition daily updates to Youtube would be particularly tricky with HPI since Google doesn't really expose nice APIs, and it's not feasible to request a takeout every day. Although once day I certainly hope to scrape it or extract from the browser cache somehow... 

Another alternative to RSS could be [Fraidycat](https://github.com/kickscondor/fraidycat), I believe it's specifically focusing content aggregation
